.github/workflows/reusable-k8s-deployment.yml

```
@@ -84,4 +84,5 @@ jobs:
             --namespace=${{ inputs.namespace_name }} \
             --create-namespace \
             --set image.tag=${{ inputs.image_tag }} \
-            --set image.repository=$container_registry_domain/$organization_domain_name/${{ inputs.app_name }}
\ No newline at end of file
+            --set image.repository=$container_registry_domain/$organization_domain_name/${{ inputs.app_name }},secrets.centops.cosmosAccount=${{ steps.appSecrets.outputs.CosmosDbAccount }},secrets.centops.cosmosKey=${{ steps.appSecrets.outputs.CosmosDbKey }}
```

This bit is a little difficult to read. Does splitting this across lines work?
```suggestion
            --set image.repository=$container_registry_domain/$organization_domain_name/${{ inputs.app_name }}, \
            secrets.centops.cosmosAccount=${{ steps.appSecrets.outputs.CosmosDbAccount }}, \
            secrets.centops.cosmosKey=${{ steps.appSecrets.outputs.CosmosDbKey }}
```